vllm/engine/llm_engine.py

```diff
@@ -707,8 +711,7 @@ def _add_processed_request(
                 encoder_seq=encoder_seq,
                 priority=priority)
         else:
-            raise ValueError(
-                "Either SamplingParams or PoolingParams must be provided.")
```

Comment:
No need to update this code now that this is the only place where `INCORRECT_PARAMS_TYPE_MSG` is used.